app/mailers/hackathon_mailer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class HackathonMailer < ApplicationMailer
+  def submission
+    @name  = params[:name]
+    @email = params[:email]
+    mail to: @email, subject: 'A new Hackathon entry was submitted!'
+  end
```

Comment:
I would say just take a `:hackathon` param and derive the name from that where needed. For the email addresses, just do all admins for now:

```ruby
email_addresses = User.admins.collect(&:email_address)
```

Reply:
Got it I'll work on it!